src/ripple/protocol/Feature.h

```diff
@@ -34,6 +34,52 @@ uint256
 feature (const char* name);
 /** @} */
 
+constexpr size_t FeatureBitsetSize = 32;
```

Comment:
The `LogicError` I was thinking of is this one: https://github.com/ripple/rippled/blob/develop/src/ripple/basics/contract.h#L74.  I'm pretty sure that it does not `throw`.  I believe it's an `assert` that happens regardless of the build flags.